content/blog/2018-09-11-birds-science.md

```diff
@@ -0,0 +1,454 @@
+metadata](/img/blog-images/2018-09-11-birds-science/wordcloud3-1.png)
+
+As expected, the words seem more focused on ecology than when looking at
+scientific papers. DataONE is a gigantic data catalogue, where one could
+
+-   study the results of such queries (e.g. meta studies of number of,
+    say, versions by datasets)
+
+-   or find data to integrate to a new study. If you want to *download*
+    data from DataONE, refer to the [download data
+    vignette](https://github.com/DataONEorg/rdataone/blob/master/vignettes/download-data.Rmd).
+
+### Conclusion
+
+In this post, we used the rOpenSci `fulltext` package, and the DataONE
+`dataone` package, to search for bird species names in scientific papers
+and scientific open datasets. We were able to draw wordclouds
+representing the diversity of topics of studies in which the birds had
+been mentioned or studied. Such a search could be fun to do for your
+favourite bird(s)! And in general, you could have more serious use cases
+of such packages.
+
+#### Scientific literature access
+
+As a reminder, the pipeline to retrieve abstracts and titles of works
+mentioning a bird species was quite smooth:
+
+``` r
+species %>%
+    tolower() %>%
+    fulltext::ft_search() %>%
+    fulltext::ft_get() %>%
+    fulltext::ft_collect() %>%
+    fulltext::ft_chunks(c("title", "abstract")) %>%
+    fulltext::ft_tabularize() %>%
+    dplyr::bind_rows()
+```
+
+`fulltext` gives you a lot of power! Other rOpenSci accessing literature
+data include [`europepmc`](https://github.com/ropensci/europepmc), R
+Interface to Europe PMC RESTful Web Service;
+[`jstor`](https://github.com/ropensci/jstor);
+[`suppdata`](https://github.com/ropensci/suppdata) for extracting
+supplemental information, and [much
+more](https://ropensci.org/packages/).
+
+#### Scientific data access… and publication with R
+
+In this post we used the [`dataone`
+package](https://github.com/DataONEorg/rdataone) to access data from
+DataONE. That same package allow uploading data to DataONE. The rOpenSci
+suite features the [`rfigshare`](https://github.com/ropensci/rfigshare)
+package for getting data from, and publishing data to,
+[Figshare](https://figshare.com/). For preparing your own data and its
+documentation for publication, check out the [`EML`
+package](https://github.com/ropensci/EML) for writing metadata
+respecting the Ecological Metadata Standard, and the [unconf `dataspice`
+project](https://github.com/ropenscilabs/dataspice) for simpler metadata
+entry.
+
+Explore more of our packages suite, including and beyond access to
+scientific literature &data and data publication,
+[here](https://ropensci.org/packages/).
+
+#### No more birding? No, your turn!
+
+This was the last post of this series, that hopefully provided an
+overview of how rOpenSci packages can help you learn more about birds,
+and can support your workflow. As a reminder, in this series we saw
+
+-   [How to identify spot for birding using open geographical
```

Comment:
"How to identify spot for birding using open geographical data."

should it be "**a** spot..."